reports/alex.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: report
+permalink: /GSoC/2017/StatusReport/AlexMaxim
+student: Alexandru-Sorin Maxim
+organisation: coala
+organisation_link : https://coala.io
+project: Improve the coala CLI
+project_link: https://summerofcode.withgoogle.com/projects/#4940869170888704
+tarball:
+mentors: >
+ [Adrian Zatreanu](https://github.com/Adrianzatreanu) & [Muhammad Kaisar Arkhan](https://github.com/yukiisbored)
+phase:
+ - Bonding : https://gitlab.com/coala/GSoC-2017/milestones/16
+ - Phase 1 : https://gitlab.com/coala/GSoC-2017/milestones/17
+ - Phase 2 : https://gitlab.com/coala/GSoC-2017/milestones/18
+ - Phase 3 : https://gitlab.com/coala/GSoC-2017/milestones/19
+bio: >
+ I'm a 3rd year student of Computer Science & Engineering at Politehnica University of Bucharest. I participated in GSoC and worked with [coala](https://coala.io) to improve coala CLI experience. Most of the people who use a CLI program do not read the whole output, only a small part of it. Also, a large output from a command could tire them and could make them stop using our application. Remove the redundant information, keeping only the necessary information.  
+social:
+ - GitHub:
+   - username: Nosferatul
+   - link: https://github.com/Nosferatul
+ - GitLab:
+   - username: Nosferatul
+   - link: https://gitlab.com/Nosferatul
+ - Gitter:
+   - username: Nosferatul
+   - link: https://gitter.im/Nosferatul
+email: alexandrus.maxim@gmail.com
+blog: https://gsocsite.wordpress.com/category/gsoc/
+activity:
+ - 0:
+   - repo: coala
+   - link: https://github.com/coala/coala/pull/4369/commits/d75d10ab7bb9be4abe7d14f201afc748fb1b96d6
+   - details: >
+      Change output for user.
+ - 1:
+   - repo: coala
+   - link: https://github.com/coala/coala/pull/4370/commits/3e590f5e34f019103f98f0ea9b32c94d01e05472
+   - details: >
+      Change the coafile warning/output.
+ - 2:
+   - repo: coala
+   - link: https://github.com/coala/coala/pull/4390/commits/2f0f3116aea7e2933322d4cc2a44a9fef61981a4
+   - details: >
+       Change the actions name.
+ - 3:
+   - repo: coala
+   - link: https://github.com/coala/coala/pull/4390/commits/8a25983187966a60a810756c200ece1d9bb59607
+   - details: Change the input: Numbers to letters
```

Comment:
Minor change, this throws an error because it interprets "Change the input" as a key because of the ":" after it.

- details : **Change the input** numbers to letters 
or 
- details: Change the input from numbers to letters will work perfect